pyproject.toml

```diff
@@ -22,6 +22,8 @@ dependencies = [
 
 [dependency-groups]
 test = [
+    "codespell>=2.4.1",
+    "greenlet>=3.2.2",
```

Comment:
Instead of installing greenlet directly, I believe we can update sqlalchemy to SQLAlchemy[asyncio].